### DISCUSSION TITLE
Idea for bugfix #1 for currency calculator

### DISCUSSION THREAD

**Discussion by SamuelTAFE** [D3nan/Tafe24S1T102_Ext#7] [General]
Hi all,
I have an idea for a potential bugfix for the currency conversion calculator.

Currently if the user forgets to select either the From or To combobox value and clicks convert the application crashes.
I had similar issue with the Fuel Conversion calc., and fixed it by setting an initial 'SelectedIndex'

Think it's also a good idea to set these starting values as different

My idea for changes are:

Add 
SelectedIndex="0"
for from combobox

Add
SelectedIndex="1"
for to combobox

We could do this in the release branch bugfixes.
Have also added screenshots for reference

Should we try the above, or does anyone else have ideas for changing this?
![Bugfix1a](https://github.com/D3nan/Tafe24S1T102_Ext/assets/162761329/e0389d56-0cc7-4459-a996-e4c5cca4ff75)
![Bugfix1b](https://github.com/D3nan/Tafe24S1T102_Ext/assets/162761329/6fdc5c1c-9a1d-4f2c-975e-9bfc7d2c0b6e)

**Comment by D3nan** [comment]
Hi Sam
Setting a default option sounds like a good idea to cover the event of the user forgetting to select one or more options. This will remove the need to deal with error handling of an empty field.

Denan
